executor/runtime/docker/docker.go

```diff
@@ -1158,6 +1161,31 @@ func (r *DockerRuntime) createTitusEnvironmentFile(c runtimeTypes.Container) err
 	return writeTitusEnvironmentFile(c.Env(), f)
 }
 
+func getUnameR() (string, error) {
+	cmd := exec.Command("/bin/uname", "-r")
```

Comment:
Can't we call uname directly from the executor? 
https://pkg.go.dev/golang.org/x/sys/unix#Utsname

Reply:
Can do.